packages/devextreme-vue-generator/src/component-generator.ts

```diff
@@ -84,6 +90,15 @@ function generate(component: IComponent, widgetsPackage: string = "devextreme",
     }
 
     namedImports.sort(compareImports);
+
+    const filterReexports = (reexports?: string[]): string[] => (
+        reexports
+          ? reexports.filter(
+            (item) => item !== "default" && (component.hasExplicitTypes && item !== "ExplicitTypes"),
```

Comment:
Can this be simplified?
```suggestion
            (item) => item !== "default" && item !== "ExplicitTypes",
```

Reply:
fixed